tools/pkglint/lib/rules.ts

```diff
@@ -664,7 +663,9 @@ export class MustDependonCdkByPointVersions extends ValidationRule {
   public readonly name = 'dependencies/cdk-point-dependencies';
 
   public validate(pkg: PackageJson): void {
-    const expectedVersion = monoRepoVersion();
+    // yes, ugly, but we have a bunch of references to other files in the repo
+    // here anyway, and it's not different than reading the root lerna.json
```

Comment:
As far as I'm concerned, stuff in `tools` is allowed to make a ton of assumptions about where it is in the mono-repo anyway.